types/mssql/mssql-tests.ts

```diff
@@ -168,11 +175,15 @@ function test_promise_returns() {
     transaction.rollback().then(() => { });
 
     var request = new sql.Request();
-    request.batch('create procedure #temporary as select * from table').then((recordset) => { });
+    request.batch('create procedure #temporary as select * from table;select 1 as value').then((recordset) => { });
     request.batch<Entity>('create procedure #temporary as select * from table;select 1 as value').then((recordset) => { });
+    request.batch`create procedure #temporary as select * from table;select ${1} as value`.then((recordset) => { });
```

Comment:
Is there a particular reason you're missing parentheses here and on lines 181, 185, and 186? It seems to be causing issues with the allowable TS version.

Reply:
Yes, for two reasons.

First, the syntax. 

    let a:string = `some string ${1}`; // some string 1

    let capture = (literals: TemplateStringsArray, ...placeholders: any[]): [TemplateStringsArray, any[]] => {
      return [literals, placeholders];
    };
    let captured: [TemplateStringsArray, any[]] = capture `some string ${1}`;
    console.log(captured); // basically - string[], any[]

Second, is that whilst the templated string syntax support has been around since v1.4, the parser could only do 

    method `template`.method()

in v > 2.8. If it were possible, I would simply say to use > 2.8 for the test file. 

The typings will work in v2.1 or later